pages/goal/volunteer/ready.js

```diff
@@ -8,32 +8,55 @@ const readyqs = [
   {
     name: 'Are you ready to volunteer?',
     description: 'Watch this video to learn what you need to know when volunteering with Voluntarily.',
-    src: 'https://www.youtube.com/embed/kv8GrLpfVNo',
+    src: 'https://www.youtube.com/embed/N0Ktkx8SYH0',
     badgeclass: 'Gp99Py5ERQGVeDFj63gizA',
```

Comment:
both videos have the same badge class - we need to create a new badge.

Reply:
Is there a way to issue a new badge class ? @avowkind